torch/csrc/dynamo/eval_frame.c

```diff
@@ -418,6 +425,8 @@ static PyObject* _custom_eval_frame(
   }
   // cache miss
 
+  // TODO(alband): This is WRONG for python3.11+ we pass in a _PyInterpreterFrame
```

Comment:
My plan for this one is to use ctypes on the python side and get the _PyInterpreterFrame directly there.
That is enough from what I've looked at in python. But splitting in a separate PR to keep this simple.

Reply:
This is done two PRs above this in the stack